.github/workflows/test.yml

```diff
@@ -4,31 +4,49 @@ on: [push, pull_request]
 
 jobs:
   run-tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
         python-version: ["3.7", "3.8", "3.9", "3.10"]
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Build packages, and install the wheel
+    - name: Install packages
+      run: |
+        pip install -U pip
+        pip install .
```

Comment:
This just tests installing from source works - I wanted to explicitly test the wheel would work.

Reply:
Which you do again later (seems redundant?)